auth0/src/main/java/com/auth0/android/Auth0.kt

```diff
@@ -188,11 +191,11 @@ public open class Auth0 @JvmOverloads constructor(
         if (url == null) {
             return null
         }
-        val normalizedUrl = url.toLowerCase()
+        val normalizedUrl = url.toLowerCase(Locale.ROOT)
```

Comment:
👍  good fix, saw this in the warnings and was going to address it, good to do it here.